godotproject/Player.gd

```diff
@@ -15,9 +15,9 @@ export (float) var air_turn_coefficient = 1.2
 
 export (float) var jump_leniency_time = 5 / 60.0
 export (float) var jump_speed = 500
-export (float) var jump_length = 10 / 60.0
+export (float) var jump_length = (10+1) / 60.0
 export (float) var shorthop_speed = jump_speed * 0.6
-export (float) var shorthop_length = 6 / 60.0
+export (float) var shorthop_length = (6+1) / 60.0
```

Comment:
I think the order between checking and adjusting was changed now that we're using a timer:
Old:
```
if jump timer is not done:
  decrease timer
  set player Y velocity to jump speed
```
New:
```
decrease timer (done by timer object)
...
if jump timer is not done:
  set player Y velocity
```
In order to keep the jump height the same, I had to add an extra frame to the jump duration.